src/chord/public/chord/chord_fwd.h

```diff
@@ -8,15 +8,15 @@
 //////////////////////////////////////////////////
 // Chord forwards
 //////////////////////////////////////////////////
-
```

Comment:
Here is an extra unnecessary line change

Reply:
You added throw to the functions so if it throws exception catching mechanism should be also implemented. At least it should not crash.